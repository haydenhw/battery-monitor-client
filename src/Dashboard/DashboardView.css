.data-reading {
    font-size: 4rem;
}

.data-card h2 {
    font-size: 1.3rem;
}

