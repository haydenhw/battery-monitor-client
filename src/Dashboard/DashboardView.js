import React from "react";

function DashboardView () {
  return (
    <div className="data-view v-view">
      <h1 className="mb-4">Battery Voltage</h1>
      <div className="data-card card mb-3">
        <div className="card-body">
          <h2>Present</h2>
          <div className="data-reading">
            <span className="v-large-font">12.6</span>
            <span>V</span>
          </div>
        </div>
      </div>
      <div className="data-card card">
        <div className="card-body">
          <h2>History</h2>
          <span>
            <img className="w-100"
                 src="https://i.picsum.photos/id/866/536/354.jpg?hmac=tGofDTV7tl2rprappPzKFiZ9vDh5MKj39oa2D--gqhA"
                 alt=""/>
          </span>
        </div>
      </div>
    </div>
    )
}

export default DashboardView
