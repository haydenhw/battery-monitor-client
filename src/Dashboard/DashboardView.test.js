import React from "react";
import renderer from 'react-test-renderer';
import DashboardView from "./DashboardView";

it('renders correctly', () => {
  const tree = renderer
    .create(<DashboardView/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
