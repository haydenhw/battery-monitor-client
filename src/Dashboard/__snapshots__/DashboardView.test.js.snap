// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="data-view v-view"
>
  <h1
    className="mb-4"
  >
    Battery Voltage
  </h1>
  <div
    className="data-card card mb-3"
  >
    <div
      className="card-body"
    >
      <h2>
        Present
      </h2>
      <div
        className="data-reading"
      >
        <span
          className="v-large-font"
        >
          12.6
        </span>
        <span>
          V
        </span>
      </div>
    </div>
  </div>
  <div
    className="data-card card"
  >
    <div
      className="card-body"
    >
      <h2>
        History
      </h2>
      <span>
        <img
          alt=""
          className="w-100"
          src="https://i.picsum.photos/id/866/536/354.jpg?hmac=tGofDTV7tl2rprappPzKFiZ9vDh5MKj39oa2D--gqhA"
        />
      </span>
    </div>
  </div>
</div>
`;
